### PR TITLE
Sent event on configurator lowlight

### PR DESCRIPTION
**Pull request by Benedito0**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-whites/issues/183 |
| Decisions | Sent an event on configurator lowlight to the manager bus  |
| Animated GIF |   |
| Dependencies |https://github.com/ripe-tech/ripe-sdk/pull/156 | |

### CODE REVIEW

File: vue/components/configurator/configurator.vue [hunk 1]
```diff
@@ -221,6 +221,14 @@ export const Configurator = {
             this.$store.commit("current_frame", frame);
         });
 
+        this.configurator.bind("highlighted_part", part => {
+            this.$bus.trigger("highlighted_part", part);
+        });
+
+        this.configurator.bind("lowlight", () => {
```
Comment (gcandal) [inline vue/components/configurator/configurator.vue]: If this event's counterpart is `highlighted_part` than this should be `lowlighted_part`.

Comment (Benedito0) [reply vue/components/configurator/configurator.vue]: That´s what I was doing but then I notice that the the lowlight event "lowlights" the whole configurator, meaning that if I went from hovering the sole to the front, even though the sole lowlights and the front highlights, the _lowlight_ event will not be triggered.

Comment (gcandal) [reply vue/components/configurator/configurator.vue]: Ok that makes sense. `lowlighted` then?

**Comment by joamag** [comment]
@gcandal @Benedito0 We don't want event pipelining of configurator instance-specific events to the bus as there may be multiple configurator instances and there's only one global bus thus it's not compatible.. @Benedito0 please explain the use case.

**Comment by gcandal** [comment]
> @gcandal @Benedito0 We don't want event pipelining of configurator instance-specific events to the bus as there may be multiple configurator instances and there's only one global bus thus it's not compatible.. @Benedito0 please explain the use case, so that we can figure out a better solution.

I totally get your point, but can't see a way to work around the problem. Our suggestion, which requires this, was https://github.com/ripe-tech/ripe-white/pull/218: allowing highlighting of the part selection line according to the highlighted part in the configurator.

On such a scenario, it's not really relevant what was the configurator where that came from. An alternative would be for the Part Selector Tech Plugin to bind itself to all configurations registered in the SDK. Do you think that would make sense?

**Comment by joamag** [comment]
@gcandal get the reference to the configurator in use or even all of them and bind for the events

**Comment by joamag** [comment]
In any way, if that's difficult let's just drop the feature, it's not that relevant

**Comment by joamag** [comment]
Found a compromise by sending the configurator as part of the event args :) https://github.com/ripe-tech/ripe-commons-pluginus/commit/3de4f43b1278cefa68733ae8e20321511b526839

**Pull request closed without merging**